persistent-test/persistent-test.cabal

```
@@ -76,7 +76,7 @@ library
       , monad-logger             >= 0.3.25
       , mtl
       , path-pieces              >= 0.2
-      , persistent               >= 2.14        && < 2.15
+      , persistent               >= 2.15        && < 2.16
```

Since there are no source changes, we can probably just do

```suggestion
      , persistent               >= 2.14        && < 2.16
```